tests/datasource_test.go

```diff
@@ -65,6 +64,14 @@ var _ = Describe("DataSource", func() {
 		f.ForceBindIfWaitForFirstConsumer(pvc)
 	}
 
+	// Delete PVC if DV was GCed, otherwise delete both
+	deletePvc := func(pvcName string) {
```

Comment:
You are potentially deleting DVs here as well, the name should probably reflect that fact.

Reply:
deletePvcAndDv? sounds bad